services/repository/src/test/java/org/sagebionetworks/repo/web/service/EntityServiceImplUnitTest.java

```diff
@@ -133,7 +118,7 @@ public void testGetFileRedirectURLForVersion() {
 	@Test
 	public void testFireCreate() {
 		when(mockUserManager.getUserInfo(PRINCIPAL_ID)).thenReturn(userInfo);
-		when(mockMetadataProviderFactory.getMetadataProvider(EntityType.project)).thenReturn(projectProviders);
+		when(mockMetadataProviderFactory.getMetadataProvider(EntityType.project)).thenReturn(Optional.ofNullable(mockProjectCreateProvider));
```

Comment:
Optional.of() would work for this case since you are guaranteed that the mock is not null.